src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionService.java

```diff
@@ -47,23 +55,46 @@ public Subscription createSubscription(Subscription subscription, String actioni
         return repository.save(subscription);
     }
 
-    public void configureListTypesForSubscription(String userId, List<String> listType) {
-        log.info(writeLog(userId, UserActions.CREATE_SUBSCRIPTION, LOCATION_ID.name()));
+    public void addListTypesForSubscription(SubscriptionListType subscriptionListType,
+                                                  String actioningUserId) {
+        log.info(writeLog(actioningUserId, UserActions.CREATE_SUBSCRIPTION, LOCATION_ID.name()));
+        //DELETE EXISTING LIST TYPE CONFIG FOR A USER
+        duplicateListTypeHandler(subscriptionListType);
```

Comment:
I think it would feel better here, rather than deleting the existing one, if we update. So logic is something like:

If present, Save, else, update

The deletion / and update of the passed in subscriptionListType here is a bit hidden, so doing like the above should make it cleaner

Reply:
Note - if agree with the comment that lists should not be configured as part of the main /subscription endpoint, then this method is no longer needed

Reply:
implemented as agreed.